README.md

```diff
@@ -244,12 +244,22 @@ Toaster users may start toaster through the **main context menu** -> **Bitbake**
 
 > Note: Toaster requires additional packages and ports. It may not be compatible with all commandWrapper settings. Read the Toaster manual to configure them to have all the required dependencies.
 
+### Linting
```

Comment:
```suggestion
### Linting
Whenever a BitBake command is run, the diagnostics are forwarded to spot syntax errors. For advanced styling recommendations, we also suggest the [oelint-adv extension](https://marketplace.visualstudio.com/items?itemName=kweihmann.oelint-vscode).

Embedded Shell and Python Languages is also supported through their respective extensions: [bash-ide](https://marketplace.visualstudio.com/items?itemName=mads-hartmann.bash-ide-vscode) and [ms-python](https://marketplace.visualstudio.com/items?itemName=ms-python.python).

Additional styling extensions can be installed and will automatically provide their diagnostics:

- [Flake8](https://marketplace.visualstudio.com/items?itemName=ms-python.flake8) (Python)
- [Pylint](https://marketplace.visualstudio.com/items?itemName=ms-python.pylint) (Python)
- [shellcheck](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck) (Shell)

See the [troubleshooting guide](./TROUBLESHOOTING.md#trade-offs-on-linting) and their respective documentation if you encounter issues with them.
```
I wanted the paragraph to be a bit more accessible for users who don't care about the inner workings.

Reply:
> Whenever a BitBake command is run, the diagnostics are forwarded to spot syntax errors.
Do you mean the diagnostics we produce from the result of the scan? It does not seem like an accurate description of what's happening. First some commands will work just fine without generating diagnostics (`open recipe's workdir` or `Open interactive terminal`) while some others just crash (`Examine recipe's dependency with taskexp`). Second, these diagnostics are not forwarded, but generated by ourselves.

Also, I don't think bash-ide and ms-python have their place in the "Linting" section. bash-ide doesn't do linting, and linting is only one functionality of ms-python.

Reply:
Now it mentions shellcheck and ms-python, but only provides a link for ms-python

Reply:
Can you fix the documentation and push it on your branch? Thanks